scripts/apiserver/resources.tmpl

```diff
@@ -42,7 +42,7 @@ type {{.res.kind}} struct {
 	{{range $field, $isNotEmpty := .res.fields}}
 		{{ $type := $field | strings.Title }}
 		{{ $varName := $.res.kind | strings.ReplaceAll "API" "Api" | strings.ReplaceAll "AWS" "Aws" }}
-		{{	$type }} {{if $isNotEmpty}} {{$varName}}{{$type}} {{else}} interface{} {{end}} `json:"{{$field | strings.ToLower}}"`
+        {{ $type}} {{if $isNotEmpty}} {{ $varName }}{{ $type }} {{else}} {{if eq $type "Owner"}} *apiv1.Owner {{else}} interface{} {{end}} {{end}} `json:"{{$field | strings.ToLower}}"`
```

Comment:
Looks like it can be of type Owner or an interface{}. I'm wondering why some of the generated types would have interface{} set instead of Owner for each one?

Reply:
The goal here was to only find code that had a var like this: Owner  interface{} within a struct and change it to this: Owner *apiv1.Owner.  This is exactly what is happening, so I'm not sure what your question is.

Reply:
I see, thanks for explaining. It was hard to read the if/else's on this line so I didn't really catch what was happening.